launch/brushless.launch

```diff
@@ -6,8 +6,8 @@
     <rosparam command="load" file="$(find mrover)/config/teleop.yaml" />
 
     <!-- run the teleop gui by starting the rosbridge server and then launching with a bash script -->
-    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"></include>
```

Comment:
is this file expected to be ran on the base station or ?? either way, add a comment and delete this ghost comment if its not important

Reply:
yeah I'm guessing the purpose of this launch file is just for testing the moteuss and is meant to be run on the pi by itself. if that's true, just clean up and fix all the outdated comments

Reply:
Yeah this is to be run on the pi, we should probably add this to the .bashrc on the pi unless we want to ssh into the pi from the jetson each time we run it.